srml/bridge/src/error.rs

```diff
@@ -16,10 +16,21 @@
 
 //! Common error types for the srml-bridge crate.
 
+use crate::Error as BridgeError;
+
 #[derive(PartialEq)]
 #[cfg_attr(feature = "std", derive(Debug))]
 pub enum Error {
```

Comment:
Oops, I forgot about the `decl_error!` error. Lets' get rid of this error type and just use that one.